ulid/_util.ts

```diff
@@ -43,16 +43,16 @@ export function incrementBase32(str: string): string {
   let charIndex;
   const maxCharIndex = ENCODING_LEN - 1;
   while (--index >= 0) {
-    char = str[index];
+    char = str[index]!;
     charIndex = ENCODING.indexOf(char);
     if (charIndex === -1) {
       throw new Error("incorrectly encoded string");
     }
     if (charIndex === maxCharIndex) {
-      str = replaceCharAt(str, index, ENCODING[0]);
+      str = replaceCharAt(str, index, ENCODING[0]!);
```

Comment:
Interesting that using `as const` on `ENCODING` at the top of this file doesn't remove the need for a non-null assertion here.